security.php

```diff
@@ -114,6 +115,7 @@ function wpcom_vip_login_limit_xmlrpc_error( $error, $user ) {
 	}
 
 	if ( is_wp_error( $login_limit_error ) ) {
+		http_response_code( 429 );
```

Comment:
FWIW, this technically will send the incorrect header to multicall requests since you could have some that succeed and others that fail. But that's very much edge case-y.

Reply:
We're already returning a rate limit error message in this case, so for practical purposes I don't think this changes anything for users?

```
return new IXR_Error( 429, $login_limit_error->get_error_message() );
```

Reply:
Yeah, this would be more of a problem for clients consuming the API since they would get an incorrect status code. But I think this is worth doing anyway (should cut down a lot of false positives we see from Scans).